Release/2.0 (#98)
SuperDARN/pydarn: README.md

As a quick tutorial on using pydarn to read a non-compressed file: 

!!! Warning 

>     pyDARN removed the IO package in the next release. Please use [pyDARNio](https://github.com/SuperDARN/pyDARNio)

Reviewer: Just to make this slightly clearer
```suggestion
    pyDARN removed the IO package from this release onwards. Please use [pyDARNio](https://github.com/SuperDARN/pyDARNio)
```

Author: Huh this is outdated

```python
import matplotlib.pyplot as plt

import pydarn

> import pydarnop

Reviewer: ```suggestion
import pydarnio
```
